common/rootfs_supervisor/etc/supervisor_scripts/common

```diff
@@ -131,3 +135,18 @@ function init_os_agent() {
     sudo os-agent &
 }
 
+
+check_bind_mount() {
+    local mount_point="$1"
+
+    if ! grep -q "$mount_point" /proc/mounts; then
+        echo ""
+        echo "WARNING: $mount_point is not a mount! This might cause issues for the Home"
+        echo "Assistant Core media and share folders mount propagation option. Make sure"
+        echo "to update your .devcontainer to the latest version."
+        echo ""
```

Comment:
Lets add a link here to the example file, we can pass in a second argument to the check_bind_mount with "supervisor" or "addon" so the correct URL can be shown.

Reply:
We don't have a devcontainer template in this repository for Supervisor. For Supervisor, it should actually just be updated by updating the Supervisor git repository. But for add-ons I can add such a message.

Reply:
Ah, right, good point 👍